app/components/auto_tag_component.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class AutoTagComponent < ViewComponent::Base
+  COLOURS = {
+    "to do": "yellow",
+    "cannot start yet": "grey",
+    "setup complete": "green",
+    "done": "green",
+  }.freeze
```

Comment:
You don't need this freeze by the way . The # frozen_string_literal: true takes care of it

Reply:
The linter added it, not me

Reply:
Surely this is freezing the hash, not the strings in it?

Or so I hope by reading it anyway. If so, the comment won't be enough to freeze the hash, I guess.

Reply:
It's a bit odd to use hash keys that have spaces in them. Pretty sure you can just use symbols

Reply:
Yes, it is freezing the hash ☹️

Reply:
they have spaces in because they're not primarily keys, they're the text that will be displayed to users - the hash is just for looking up what colour each text should have